doc/source/contributing/doc_dev_summary.md

```diff
@@ -0,0 +1,51 @@
+(doc_dev_summary)=
+# Building the documentation
+
+The preferred workflow for contributing to ArviZ is to fork
+the [GitHub repository](https://github.com/arviz-devs/arviz/),
+clone it to your local machine, and develop on a feature branch. For a detailed description of the recommended development process, see the step-by-step guide {ref}`here <build_steps>`.
+
+If you are using Docker, see {ref}`building_doc_with_docker`.
+
+## Pull request checks
+
+Every PR has a list of checks that check if your changes follow the rules being followed by ArviZ docs.
+You can check why a specific test is failing by clicking the `Details` next to it. It will take you to errors and warning page. This page shows the details of errors, for example in case of docstrings:
+
+`arviz/plots/pairplot.py:127:0: C0301: Line too long (142/100) (line-too-long)`.
+
+It means line 127 of the file `pairplot.py` is too long.
+
+### Docstring
+
+Docstrings follow the pylint rules. Don't forget to check the following points before opening your PR:
```

Comment:
docstrings follow numpydoc first and foremost. Some of the numpydoc constraints are checked by `pydocstyle` (it is also a check in azure), then as content of `.py` files docstrings are also subject to pylint and black rules which include those line too long and trailing whitespaces that are arguably the easiest ones to fail. But those are not the only checks nor the most important ones.